Fluid.MvcSampleVB/Program.vb

```diff
@@ -0,0 +1,16 @@
+Imports System
+Imports System.IO
+Imports Microsoft.AspNetCore.Hosting
+
+Module Program
+    Sub Main(args As String())
+        Dim host = New WebHostBuilder() _
```

Comment:
I don't think the line continuation is needed here

Reply:
Do you mean that the newline format should be kept except without the `_`, or should it all be done all on one line?

Reply:
Nope, VB multi line continuation, in declaration case you can try this  
```vb
Call New WebHostBuilder().
            UseKestrel().
            UseContentRoot(Directory.GetCurrentDirectory()).
            UseIISIntegration().
            UseStartup(Of Startup)().
            Build().
            Run()
```

Reply:
Or you can use `With` statement if you do so

Reply:
Sweet, Didn't know about the `.` on the end of a line for it to continue!